cmd/nginx-ingress/main.go

```diff
@@ -79,7 +79,7 @@ func main() {
 		appProtectVersion = getAppProtectVersionInfo()
 	}
 
-	updateSelfWithVersionInfo(kubeClient, version, nginxVersion, appProtectVersion)
+	updateSelfWithVersionInfo(kubeClient, version, nginxVersion.String(), appProtectVersion)
```

Comment:
is its possible to check if pod labels are being assigned correctly after this change?

Reply:
Not sure if we have a test for it, but locally seems fine: `app.nginx.org/version: 1.25.3-nginx-plus-r31`